mdx/guides/integrate-instant.mdx

```diff
@@ -45,33 +54,35 @@ The 0x Instant UI and Asset Swapper are bundled together in a convenient JS pack
-| `networkId`                      | Id of Ethereum network to connect to. Defaults to 1 (mainnet).                                                                                                                                                                                                                                                                                       |
-| `affiliateInfo`                  | An object specifying what % ETH fee should be added to orders and where the fee should be sent. Max feePercentage is .05 (See examples below)                                                                                                                                                                                                        |
-| `shouldDisableAnalyticsTracking` | An option to turn on / off analytics used to make Instant a better experience. Defaults to false.                                                                                                                                                                                                                                                    |
-| `onSuccess` | a function handler that is called when the token purchase through 0x Instant is complete. The function handler requests one argument: the transaction hash.                                                                                                                                                                                                                                                                                       |
-| `onClose` | a function handler that is called when the Instant overlay is closed. The function handler does not request any argument                                                                                                                                                                                                                                                                                       |
+| `chainId`                        | Chain Id of Ethereum network to connect to. Defaults to 1 (mainnet).                                                                                                                                                                                                                                                                                                                                                                                  |
+| `affiliateInfo`                  | An object specifying what % ETH fee should be added to orders and where the fee should be sent. Max feePercentage is .05 (See examples below)                                                                                                                                                                                                                                                                                                         |
```

Comment:
I think this might be gone now? What ended up happening with this?

Reply:
We haven't removed this at all. The calculation is currently correct, but not optimal.